src/folderlistmodel.cpp

```diff
@@ -419,7 +427,9 @@ void FolderListModel::resetModel()
     }
     // Outbox
     FolderItem *item = new FolderItem(QModelIndex(), QMailFolder::LocalStorageFolderId, OutboxFolder,
-                                      QMailMessageKey::status(QMailMessage::Outbox), 0);
+                                      QMailMessageKey::status(QMailMessage::Outbox) &
+                                      ~QMailMessageKey::status(QMailMessage::Trash) &
```

Comment:
Trash flag was missing here, in theory it will never have those since this is our local flag and we should never send messages with this flag to standard folder outbox, added to make things more consistent.